commands/FBDelay.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/python
+import string
+import lldb
+import fblldbbase as fb
+import fblldbobjcruntimehelpers as runtimeHelpers
+from threading import Timer
+
+
+def lldbcommands():
+  return [
+    FBDelay()
+  ]
+
+class FBDelay(fb.FBCommand):
+  def name(self):
+    return 'zzz'
+
+  def description(self):
+    return 'Executes specified lldb command after delay.'
+
+  def options(self):
+    return []
+
+  def args(self):
+    return [
+      fb.FBCommandArgument(arg='delay in seconds', type='float', help='time to wait before executing specified command'),
+      fb.FBCommandArgument(arg='lldb command', type='string', help='another lldb command to execute after specified delay', default='process interrupt')
+    ]
+
+  def run(self, arguments, options):
+    lldb.debugger.SetAsync(True)
```

Comment:
I actually don't see it being used like that in [other cases](https://github.com/facebook/chisel/search?utf8=✓&q=lldb.debugger.SetAsync&type=Code) we either enable reset async conditionally or don't do it at all in [FBFindCommands.py](https://github.com/facebook/chisel/blob/c3d7e95ebdbae03e246bcb56e19da77fddea5e16/commands/FBFindCommands.py#L119)

I don't mind resetting if you are certain we should do it. 
Sadly, no documentation, and I don't feel like lldb spelunking right now 😳